Tools/LyTestTools/ly_test_tools/o3de/editor_test_utils.py

```diff
@@ -71,7 +71,15 @@ def retrieve_crash_output(run_id: int, workspace: AbstractWorkspaceManager, time
     :return str: The contents of the editor crash file (error.log)
     """
     crash_info = "-- No crash log available --"
-    crash_log = os.path.join(retrieve_log_path(run_id, workspace), 'error.log')
+    error_log_regex = ""
+    log_path = retrieve_log_path(run_id, workspace)
+    # Gather all of the files in the log directory
+    dir_files = [f for f in os.listdir(log_path) if os.path.isfile(os.path.join(log_path, f))]
+    for file_name in dir_files:
+        # Search for all .log files with either "crash" or "error" because they could be renamed
```

Comment:
what does this mean with "renamed"?

Reply:
I'm not sure what is exactly renaming the log files, but we've seen error.log files being renamed i.e. error_2021_11_15_12_26_33.log

Reply:
those are backed up log files from previous runs. Don't search in these. The last ran test will always be error.log

Reply:
Thanks, found actual issue and submitted new changes